### PR TITLE
Stack trace origin

### PR DESCRIPTION
**Pull request by kasperl**

### CODE REVIEW

File: tests/negative/gold/illegal_system_call_test.gold [hunk 1]
```diff
@@ -0,0 +1,9 @@
+As check failed: null is not a ServiceResource.
```
Comment (kasperl) [inline tests/negative/gold/illegal_system_call_test.gold:1]: Before this change, this test would have generated a complete nonsense stack trace.

File: src/compiler/toitc.cc [hunk 2]
```diff
@@ -98,225 +76,211 @@ int main(int argc, char **argv) {
...
-    }
-    char* image_filename = argv[2];
-    char* bundle_filename = argv[3];
-    auto bundle = SnapshotBundle::read_from_file(bundle_filename);
-    if (!bundle.is_valid()) print_usage(1);
-    write_image_from_bundle(image_filename, bundle);
-    free(bundle.buffer());
-  } else {
-    char* bundle_filename = null;
+  char* bundle_filename = null;
```
Comment (kasperl) [inline src/compiler/toitc.cc:79]: Only indentation changed below here.

File: tests/containers_test.toit [hunk 3]
```diff
@@ -7,7 +7,8 @@ import expect show *
 
 main:
   images/List := containers.images
-  expect_list_equals [containers.current] images
+  expect_equals 2 images.size
```
Comment (kasperl) [inline tests/containers_test.toit:10]: Not really sure this is the behavior I want, but for now we do return the system image in the list of containers.

**Review by floitsch — APPROVED** [review]
LGTM.

**Pull request merged**